webcam.go

```diff
@@ -86,6 +86,16 @@ func (w *Webcam) GetSupportedFormats() map[PixelFormat]string {
 	return result
 }
 
+// GetName returns the human-readable name of the device
+func (w *Webcam) GetName() (string, error) {
+	return getName(w.fd)
+}
+
+// GetBusInfo returns the location of the device in the system
+func (w *Webcam) GetBusInfo() (string, error) {
```

Comment:
should this have a better name in this case? Like path or something? the syscall side is fine but this file is an opportunity for cleaner names